src/Megadraft.js

```diff
@@ -14,6 +14,8 @@ import Sidebar from "./components/Sidebar";
 import Media from "./components/Media";
 import DEFAULT_PLUGINS from "./plugins/default";
 import DEFAULT_ACTIONS from "./actions/default";
+import DEFAULT_MEDIA_FEATURED from "./components/plugin/featuredOptions";
```

Comment:
Could it be something like:

`import DEFAULT_FEATURED_OPTIONS from "./components/plugin/defaults";`

Or:

`import FEATURED_OPTIONS from "./components/plugin/defaults";`